README.md

```diff
@@ -53,6 +53,23 @@ Logging is preferred through a library, avoiding printf, console.log, etc. Loggi
 - [Elastic Common Schema](https://www.elastic.co/guide/en/ecs-logging/overview/current/intro.html)
 - [Sentry](https://sentry.io/)
 
+### Key Principles of Secure Logging
+
+- **Avoid Logging Sensitive Data**: Ensure that logs do not contain sensitive information such as passwords, email addresses, or PII. Scrubbing or masking such data is crucial to maintaining privacy and security.
```

Comment:
Would suggest to not use abbreviations to make it more readable.
So Personal Identifiable Information (PII)